src/modules/fetchcomment.js

```diff
@@ -0,0 +1,15 @@
+const commentsload = document.querySelector('.commentsload');
+
+export default async function fetchcomment(id, key) {
+  commentsload.innerHTML = '';
+  await fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${key}/comments?item_id=${id}`)
+    .then((res) => res.json())
+    .then((output) => {
+      if (output.error) {
+        return;
+      }
+      output.forEach((comment) => {
+        commentsload.innerHTML += `<p><span>${comment.creation_date} ${comment.username}: ${comment.comment}</span></p>`;
+      });
+    });
+}
```

Comment:
- [ ] Same here, you should use `arrow` syntax for functions instead of JavaScript Function Expressions because in this project you learn how to use ES6 new features. Kindly make this :wink: 💯 

- Your code might looks like:


```js
const commentsload = document.querySelector('.commentsload');

const fetchcomment = async (id, key) => {
  commentsload.innerHTML = '';
  await fetch(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/${key}/comments?item_id=${id}`)
    .then((res) => res.json())
    .then((output) => {
      if (output.error) {
        return;
      }
      output.forEach((comment) => {
        commentsload.innerHTML += `<p><span>${comment.creation_date} ${comment.username}: ${comment.comment}</span></p>`;
      });
    });
};

export default fetchcomment;
```

Reply:
done